### PR TITLE
test: add helper for handling windows newlines on CI

### PR DESCRIPTION
**Pull request by derberg**
**Description**

- add helper for handling windows newlines on CI

**Related issue(s)**
See also https://github.com/asyncapi/converter-js/pull/29/checks?check_run_id=1869812214

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -8,104 +8,112 @@ describe('#convert', () => {
...
-        assert.strictEqual(output, result);
+        assert.strictEqual(removeLineBreaks(output), removeLineBreaks(result));
     });
 
     it('should convert from 2.0.0-rc2 to 2.0.0', () => {
         const input = fs.readFileSync(path.resolve(__dirname, 'input', '2.0.0-rc2', 'streetlights.yml'), 'utf8');
         const output = fs.readFileSync(path.resolve(__dirname, 'output', '2.0.0', 'streetlights.yml'), 'utf8');
         const result = convert(input, '2.0.0');
-        assert.strictEqual(output, result);
+        assert.strictEqual(removeLineBreaks(output), removeLineBreaks(result));
```
Comment (magicmatatjahu) [inline test/index.js:109]: you can put put assert to common function like:

```js
function AssertStrings(output, result) {
  assert.strictEqual(removeLineBreaks(output), removeLineBreaks(result));
}
```

**Review by magicmatatjahu — APPROVED** [review]

**Review by magicmatatjahu — APPROVED** [review]

**Comment by derberg** [comment]
@magicmatatjahu yeah just did, approve again please

**Comment by derberg** [comment]
@magicmatatjahu tbh this error from sonarcube doesn't make sense now, it tests. If about would abstract those tests into one then those tests would be hard to read. I suggest to ignore duplication rule in sonar in test dir as we did in other repos. Agreed?

**Comment by magicmatatjahu** [comment]
@derberg you should have possibility to ignore solar in `tests` folder like here https://github.com/asyncapi/parser-js/blob/master/.eslintrc#L103

**Comment by derberg** [comment]
@magicmatatjahu will do it in the sonar UI as we do not have linter in converter yet 😅 
just approve and I'm on it

**Review by magicmatatjahu — APPROVED** [review]

**Comment by magicmatatjahu** [comment]
@derberg go ahead :)

**Pull request merged**